Update collaboration-avatar-group.asciidoc (#133)
vaadin/docs: articles/ce/collaboration-avatar-group.asciidoc

of navigating to another view or closing the browser tab, the user's avatar is
removed from the group.

.Example of a `CollaborationAvatarGroup`

> image::images/collaboration-avatar-group-example.png[Example of a `CollaborationAvatarGroup`]

Reviewer: ```suggestion
image::images/collaboration-avatar-group-example.png[Three avatars in a group, with three more avatars collapsed to an overflow item]
```
As general guidance, the alt text of an image should try to describe what the image contains. The original text is probably sufficient as well, but I wanted to try and improve it a bit, and not repeat the image title.

Side note: why are two avatars in the image using the same color (BS and the sheep)?

Author: Not sure exactly how it should behave. In our demo, if you have color green, when editing one entity in the list, then you will have green after switching to another entity as well. If you switch over to another entity where someone already has the green color assigned, then there will now be two persons with green.

UserInfo userInfo = new UserInfo(userEntity.getId());

CollaborationAvatarGroup avatarGroup = new CollaborationAvatarGroup(
        userInfo, "ensuranceClaims");
add(avatarGroup);
----

> .`User`, `UserService` and "ensuranceClaims" are examples

Reviewer: ```suggestion
.`User`, `UserService` and `"ensuranceClaims"` are examples
```
I would keep this consistent, that all three of them are code snippets

[NOTE]

> `User` and `UserService` are examples of classes that might exist in your application for user management, and they are not available in Collaboration Engine.
> "ensuranceClaims" is the topic id, which can be any string to identify what the users are collaborating on.

Reviewer: ```suggestion
The `User` and `UserService` classes are not available in Collaboration Engine. They are examples of classes that might exist in your application for user management.
The topic ID `"ensuranceClaims"` can be any string to identify what the users are collaborating on.
```
I would try to avoid starting a sentence with either quotes or lowercase word/code snippet.

Author: I updated the suggestion to add a line break on row 39, as it now contained two sentences.
